nixos/modules/programs/dublin-traceroute.nix

```
@@ -0,0 +1,31 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+  cfg = config.programs.dublin-traceroute;
+
+in {
+  meta.maintainers = pkgs.dublin-traceroute.meta.maintainers;
+
+  options = {
+    programs.dublin-traceroute = {
+      enable = mkEnableOption (mdDoc ''
+      dublin-traceroute, add it to the global environment and configure a setcap wrapper for it.
+      '');
+
+      package = mkPackageOptionMD pkgs "dublin-traceroute" { };
```

Now that we're past 23.05, this can probably be changed back to it's non-MD alias.

```suggestion
      package = mkPackageOption pkgs "dublin-traceroute" { };
```

---

Suggestions like these i see are quite common, but these fixes are trivial to apply treewide. Until `mkPackageOptionMD` is removed I wouldn't worry about this.

---

Sorry about that, I did not intend to block the PR.